lib/applications/resolvers.ts

```diff
@@ -661,7 +661,7 @@ export const generateApplicationsReport: Resolver = async (_, args, { prisma })
     return {
       ...application,
       dateOfBirth: formatDate(application.dateOfBirth),
-      applicationDate: application.createdAt.toLocaleDateString('en-US', {
+      applicationDate: application.createdAt?.toLocaleDateString('en-US', {
```

Comment:
Unrelated to this PR but ran into an error where exporting the applications requests csv would fail if application date checkbox wasn't selected
![image](https://user-images.githubusercontent.com/51224641/148309148-5067478d-aaf9-4c4d-86d4-ad0986484cbe.png)